### PR TITLE
New NOT_WORKING machines

### PR DESCRIPTION
**Pull request by clawgrip**
-----------------------------
Photo Play 2002 (Belgium) [Asure]

New NOT_WORKING clones
-----------------------------
Photo Play 1999 (Netherlands) [Asure]
Photo Play 2000 (Netherlands) [Asure]
Photo Play 2001 (Netherlands) [Asure]
Photo Play Masters 2001 (Netherlands) [Asure]

### CODE REVIEW

File: src/mame/drivers/photoply.cpp [hunk 1]
```diff
@@ -387,6 +388,21 @@ ROM_START(photoply99sp)
 	DISK_IMAGE( "photoplay99sp", 0, BAD_DUMP SHA1(887e5b8c931d6122a1c3a8eda5cb919eb162eced) ) // From an operated HDD. A clean one must be recreated from the CDs
 ROM_END
 
+// BIOS not provided on this set
+ROM_START(photoply99nl)
+	ROM_REGION(0x20000, "bios", 0) // Motherboard BIOS
+	ROM_LOAD("funworld_award_486e_w83787.bin", 0x000000, 0x20000, CRC(af7ff1d4) SHA1(72eeecf798a03817ce7ba4d65cd4128ed3ef7e68) ) // 486E 96/7/19 W83787 PLUG & PLAY BIOS, AT29C010A
```
[the remark this answers is not in the document]
Comment (clawgrip) [reply src/mame/drivers/photoply.cpp]: You're right. Added.

File: src/mame/drivers/photoply.cpp [hunk 2]
```diff
@@ -290,6 +290,7 @@ void photoply_state::photoply(machine_config &config)
 		.erase_all_time(attotime::from_usec(10));
 }
 
+// We asume that every Photo Play from 1999 onwards use a DX4 100MHz instead of a 75MHz one (both were compatible, the later were recommended)
```
Comment (cuavas) [inline src/mame/drivers/photoply.cpp:293]: “Later” and “latter” are not the same word.

File: src/mame/drivers/photoply.cpp [hunk 3]
```diff
@@ -301,10 +302,10 @@ ROM_START(photoply98sp)
 	ROM_LOAD("funworld_award_486e_w83787.bin", 0x000000, 0x20000, CRC(af7ff1d4) SHA1(72eeecf798a03817ce7ba4d65cd4128ed3ef7e68) ) // 486E 96/7/19 W83787 PLUG & PLAY BIOS, AT27C010, Funworld sticker: Sept 1998
 
 	ROM_REGION(0x8000, "ex_bios", ROMREGION_ERASE00 ) // Multifunction board with a ESS AudioDrive chip, Funworld sticker: Sept 1998
-	ROM_LOAD("enhanced_bios_centos.bin", 0x000000, 0x8000, CRC(ee8ad003) SHA1(4814385117599a98da02155785d1e3fce4e485bd) ) // Centos CI-8000/PP2000 ROM BIOS Version 1.06, 27C256B
+	ROM_LOAD("enhanced_bios_centos.bin", 0x0000, 0x8000, CRC(ee8ad003) SHA1(4814385117599a98da02155785d1e3fce4e485bd) ) // Centos CI-8000/PP2000 ROM BIOS Version 1.06, 27C256B
 
 	ROM_REGION(0x8000, "video_bios", 0 )
-	ROM_LOAD("cl-gd5446_pci_vga_bios_version_1.31.u2", 0x0000, 0x8000, CRC(61f8cac7) SHA1(6e54aadfe10dfa5c7e417a054e9a64499a99083c) ) // Cirrus Logic/Quadtel CL-GD5446 PCI VGA BIOS v1.31 , AT27C256R
+	ROM_LOAD("cl-gd5446_pci_vga_bios_version_1.31.u2", 0x0000, 0x8000, CRC(61f8cac7) SHA1(6e54aadfe10dfa5c7e417a054e9a64499a99083c) ) // Cirrus Logic / Quadtel CL-GD5446 PCI VGA BIOS v1.31 , AT27C256R
```
Comment (cuavas) [inline src/mame/drivers/photoply.cpp:308]: Conventional English style does not place spaces around a solidus (slash).

**Review by cuavas — COMMENTED** [review]
I’d question whether update/recovery discs should be in the machines.  They’re probably more appropriate for a software list, in the same way that we have hash/gtfore.xml for Golden Tee Fore! update discs.

File: src/mame/drivers/photoply.cpp (hunk 2 above)
Comment (clawgrip) [reply src/mame/drivers/photoply.cpp]: Fixed, thanks

File: src/mame/drivers/photoply.cpp (hunk 3 above)
Comment (clawgrip) [reply src/mame/drivers/photoply.cpp]: Fixed, thx

**Comment by clawgrip** [comment]
> I’d question whether update/recovery discs should be in the machines. They’re probably more appropriate for a software list, in the same way that we have hash/gtfore.xml for Golden Tee Fore! update discs.

I raised the same question on Discord (@smf- , @galibert ), but there was no authoritative and clear answer to it. Anyway, this PR isn't related to the recovery discs.

**Comment by smf-** [comment]
I would make everyone go through an initial install using the discs as
it's the only way they get tested.

File: src/mame/drivers/photoply.cpp [hunk 4]
```diff
@@ -491,9 +604,14 @@ ROM_START(photoply2k4)
...
+GAME( 1999, photoply99nl,    photoply99sp,  photoply_dx4_100, photoply, photoply_state, empty_init, ROT0, "Funworld", "Photo Play 1999 (Netherlands)",         MACHINE_NOT_WORKING|MACHINE_NO_SOUND|MACHINE_UNEMULATED_PROTECTION )
+GAME( 2000, photoply2k,      0,             photoply_dx4_100, photoply, photoply_state, empty_init, ROT0, "Funworld", "Photo Play 2000 (v2.01)",               MACHINE_NOT_WORKING|MACHINE_NO_SOUND|MACHINE_UNEMULATED_PROTECTION )
+GAME( 2000, photoply2ksp,    photoply2k,    photoply_dx4_100, photoply, photoply_state, empty_init, ROT0, "Funworld", "Photo Play 2000 (Spain)",               MACHINE_NOT_WORKING|MACHINE_NO_SOUND|MACHINE_UNEMULATED_PROTECTION )
+GAME( 2000, photoply2knl,    photoply2k,    photoply_dx4_100, photoply, photoply_state, empty_init, ROT0, "Funworld", "Photo Play 2000 (Netherlands)",         MACHINE_NOT_WORKING|MACHINE_NO_SOUND|MACHINE_UNEMULATED_PROTECTION )
+GAME( 2001, photoply2k1it,   0,             photoply_dx4_100, photoply, photoply_state, empty_init, ROT0, "Funworld", "Photo Play 2001 (Italia)",              MACHINE_NOT_WORKING|MACHINE_NO_SOUND|MACHINE_UNEMULATED_PROTECTION )
```
Comment (angelosa) [inline src/mame/drivers/photoply.cpp:613]: This should be `(Italy)`, unless it's sold verbatim as `Photo Play 2001 Italia`

**Comment by cuavas** [comment]
But, at least according to the labels you've given them, this PR *does* add **update** discs.  Aren’t these discs used to update a system, not media the game runs from directly?

File: src/mame/drivers/photoply.cpp (hunk 4 above)
Comment (clawgrip) [reply src/mame/drivers/photoply.cpp]: Fixed, thx

File: src/mame/arcade.flt [hunk 5]
```diff
@@ -1222,6 +1222,7 @@ spaceg.cpp
 spartanxtec.cpp
 spbactn.cpp
 spcforce.cpp
+spdamges.cpp
```
Comment (Osso13) [inline src/mame/arcade.flt:1225]: This doesn't match the name of the driver (spdamjes.cpp). And same in mame.lst.

Comment (clawgrip) [reply src/mame/arcade.flt]: I used the wrong branch, it's an unrelated work, I've reverted the commit.

**Comment by cuavas** [comment]
> There's no CD-ROM attached in driver, and it needs one to make a SW list to work properly.

Yeah, but it needs a CD-ROM drive attached for CD images in the system ROM definitions to do anything useful, too.  Having the CDs as part of the system like this doesn’t represent how the systems were actually used.

**Comment by angelosa** [comment]
In m55hipl driver photoply2k1mtnl boots up to checking COM3 port for the microtouch


There's also a Microcal tester (among others) in the image, sounds important to have this.

**Pull request merged**